hack/update-mocks.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
```

Comment:
![image](https://user-images.githubusercontent.com/17962021/196841917-fc1170cc-0502-44b9-a29b-bdcc08db2884.png)

Reply:
In my local environment.

Reply:
Ops, I will chmod it